src/main/resources/application.properties

```diff
@@ -6,3 +6,4 @@ spring.cloud.discovery.enabled = true
 server.port = 3005
 spring.data.mongodb.database=user_db
 spring.data.mongodb.port=27017
+logging.file.name=D:/Spring/Assessment/user-service/src/logs/user_service.log
```

Comment:
So your code will logs to your local only please avoid this and try to give relative path